packages/connection-info/src/connection-info.ts

```diff
@@ -12,6 +12,8 @@ export interface AtlasClusterMetadata {
   clusterName: string;
   clusterType: 'host' | 'replicaSet' | 'cluster' | 'serverless';
   regionalBaseUrl: string;
+  // TODO: maybe instanceSize?
+  supportsRollingIndexes?: boolean;
```

Comment:
The tech design named it `supportsRollingIndexCreation` which is probably better.

Reply:
I removed it in place of just storing instanceSize here. We'll go with the feature rollingIndexCreation or a dedicated hook useSupportsRollingIndexCreation.